lib/screenplay_web/controllers/pa_messages_api_controller.ex

```diff
@@ -56,4 +58,17 @@ defmodule ScreenplayWeb.PaMessagesApiController do
       |> json(%{error: "not_found"})
     end
   end
+
+  defp log_pa_message(event, message, conn) do
+    Screenplay.Util.log(event,
```

Comment:
I know it isn't called out in the task, but should we include `alert_id`? Just want to mention it in case it was an oversight.